packages/web-integration/src/common/tasks.ts

```diff
@@ -213,17 +214,7 @@ export class PageTaskExecutor {
                   );
                   // clear the input field
                   // Select all content in the input field
-                  //TODO: This needs to be abstracted to meet the mobile scene
-                  const isMac = process.platform === 'darwin';
-                  if (isMac) {
-                    await this.page.keyboard.down('Meta');
-                    await this.page.keyboard.press('a');
-                    await this.page.keyboard.up('Meta');
-                  } else {
-                    await this.page.keyboard.down('Control');
-                    await this.page.keyboard.press('a');
-                    await this.page.keyboard.up('Control');
-                  }
+                  await this.page.selectAll();
                   await this.page.keyboard.press('Backspace');
```

Comment:
@zhoushaw It is recommended that these be fully encapsulated without exposing atomic operations such as keyboard?

Reply:
LGTM